app/controllers/comment_controller.php

```diff
@@ -1,7 +1,71 @@
 <?php
 	class CommentController extends AppController
 	{
+		public function write()                        
+	    {
+	        $thread = Thread::get(Param::get('thread_id'));
+	        $threadId = Param::get('thread_id');
+	        $comment = new Comment;
+	        $page = Param::get('page_next', 'write');
+	        switch ($page) {
+	        case 'write':                    
+	            break;
+	        case 'write_end':                
+	            $comment->username = Param::get('username');
```

Comment:
Please avoid using magic strings (Apply to all) :bow:
Similar to magic numbers, you may declare these as constant variables.

```
const WRITE = 'write';
const WRITE_END = 'write_end';
...
case self::WRITE:
    break;
case self::WRITE_END:
      $comment->username = Param::get('username');
```